docs/user-guide/FAQ.md

```diff
@@ -23,6 +25,14 @@ toc:
 
 # Frequently Asked Questions
 
+## How do I skip a build?
+
+You might want to skip a build if you're only changing documentation.
+
+You can add `[ci skip]` or `[skip ci]` somewhere in the commit message. Adding this message will cause Screwdriver to ignore that commit while determining whether to start a build. If you don't want Screwdriver to trigger a build when you merge the pull request, add `[ci skip]` or `[skip ci]` to each commit in the pull request as well as at the the beginning of the title of the pull request.
```

Comment:
if the [ci skip] thing is in every commit, then there is no need for it to be in the title right?